src/components/TextButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export interface textButtonInterface {
+	text: string;
+	onClick: () => void;
+}
+
+const ButtonStyle = styled.div`
+	color: #bdbdbd;
+	font-family: Montserrat, Arial;
+	font-size: 14px;
```

Comment:
Let's do this ( Don't apply the change in github )
```suggestion
	font-size: 1em;
```

Reply:
Shoot, don't know why this was changed back to px; thanks for pointing it out :^)